README.md

```diff
@@ -19,74 +16,16 @@
 
 ## Install
 
-#### For ESLint 5.x
-
-To install simply add this config (along with ESLint and required dependencies) to your project dev-dependencies:
-
 ```
-npm install --save-dev babel-eslint
-npm install --save-dev eslint
 npm install --save-dev eslint-config-vazco
-npm install --save-dev eslint-plugin-babel
-npm install --save-dev eslint-plugin-react
-npm install --save-dev eslint-plugin-vazco
-
-# If you want to use prettier
-npm install --save-dev prettier
-npm install --save-dev eslint-plugin-prettier
-npm install --save-dev eslint-config-prettier
 ```
 
 Ready to use one-liner (for `default` config):
```

Comment:
There is no need for an extra one liner, when the main command also is in one line :)

Reply:
Done.